### PR TITLE
avoid pie chart's label out of range

### PR DESCRIPTION
**Pull request by airwin**
avoid label of the pie chart out of range by the `avoidLabelOverlap`

### CODE REVIEW

**Comment by lianliu22** [comment]
Thank you for your fix.

BTW, do you know what the process of merging a PR is?

**Comment by airwin** [comment]
@lianliu22 I think an origin PR should fork&edit&make a PR. For this quick PR, I just click the edit button of the file in the official repo, and follow the github's guide. I don't know if there's something improper..

**Comment by Ovilia** [comment]
Thanks for your contribute. I'll review the code later.

**Comment by PanJiaChen** [comment]
Is there any new progress?

**Comment by Ovilia** [comment]
@PanJiaChen Sorry for the slow reply. We will review this soon.

File: src/chart/pie/labelLayout.js [hunk 1]
```diff
@@ -28,18 +28,20 @@ function adjustSingleSide(list, cx, cy, r, dir, viewWidth, viewHeight) {
 
     // 压
     function shiftDown(start, end, delta, dir) {
+        // 防止边缘出界
+        var couldShiftUp = (list[0].y - delta / 2) >= list[0].height * 0.61;
```
Comment (Ovilia) [inline src/chart/pie/labelLayout.js]: What does 0.61 mean?

**Comment by Xianbei233** [comment]
@airwin

**Pull request closed without merging**